fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Table.java

```diff
@@ -47,11 +49,20 @@ public class Table extends BaseObject {
     // The column to use when making this table multi-tenant (if supported by the the target)
     private final String tenantColumnName;
 
+    // The rules to distribute the table in a distributed RDBMS implementation (Citus)
+    private final DistributionType distributionType;
+
+    // If set, overrides the column used to distribute the data in a distributed database
+    private final String distributionColumnName;
+
     // The With parameters on the table
     private final List<With> withs;
     
     private final List<CheckConstraint> checkConstraints = new ArrayList<>();
 
+    // Do we still want to create this table?
+    private final boolean create;
```

Comment:
a little confused about when/where we'd want this.  i guess for deleting tables?

Reply:
Yes, it's required when we need to delete a table. There are a number of places where we need to skip processing if the table has been dropped.